src/lib/util/cdrom.cpp

```diff
@@ -1552,6 +1555,81 @@ static const uint16_t qoffsets[ECC_Q_NUM_BYTES][ECC_Q_COMP] =
 };
 
 
+/**
+ * @brief   -------------------------------------------------
+ *            EDC_crctable - each value represents a CRC used
+ *            in calculating the EDC for MODE1 sectors
+ *          -------------------------------------------------.
+ */
+
+static const uint32_t EDC_crctable[256] =
+{
+	0x00000000, 0x90910101, 0x91210201, 0x01B00300,
```

Comment:
What’s the polynomial for this CRC?  It would be easier to understand if the table was generated using the CRC polynomial.  Also, we prefer lowercase hex literals.

Reply:
resolved in eb23b45421bf94e4a075ddedc4bb5879231bfe16 and e8d88259b0d881b29c18a19c480c04caf210f517

Reply:
You’ve still got the table there, you’re not generating it using the polynomial.

Reply:
resolved in dde21b19362459afbfa811f74b35c523c7ef0d35